liquibase-core/src/test/java/liquibase/database/core/PostgresDatabaseTest.java

```diff
@@ -123,4 +131,64 @@ public void test_getConcatSql() {
         assertEquals("one || two || | three", database.getConcatSql("one", "two", "| three"));
     }
 
+    @Test
+    public void generatePrimaryKeyName_tableSizeNameLessThan63Bytes_nameIsBuiltCorrectly() {
```

Comment:
Let's create a private helper method:

```
    @Test
    public void assertPrimaryKeyName(final String tableName, final String expectedPrimaryKeyName) {
        // Act
        final String pk = this.database.generatePrimaryKeyName(tableName);

        // Assert
        assertEquals(expectedPrimaryKeyName, pk);
        assertTrue(expectedPrimaryKeyName.getBytes(CHARSET).length <= PGSQL_PK_BYTES_LIMIT);
    }
```

Now you can call this from all of your tests.

Reply:
Basically agree except `assertPrimaryKeyName` should accept actual PK instead of table name.
That's clearer to pass both actual and expected names instead of table name.
And it's clearly shows the tested method call inside the test. 
I'd vote for explicitness instead of shortness here.